dataservice/__init__.py,sha256=_NWjtHnQwCEXmjUeLeZ4cSrQaYA4T3s9snoKzF8WAGU,740
dataservice/_utils.py,sha256=Pmdo-n1vt9VJImjQopVSxkMnYavpfDqms9BCg5WceFU,600
dataservice/cache.py,sha256=qK_vXJ6ucXRtRAFH4zyItjjV1jBO5pZg5BMwh_B_y8A,3641
dataservice/clients.py,sha256=IgZW7_mYI2K-mkzMyVDVig3xnLy3lOC9UZ2d5Akubg0,3607
dataservice/config.py,sha256=YScQkLcuW_X_WEcbd1cJHR5Kl26eSWyZPDQM2rSE1PQ,2316
dataservice/data.py,sha256=ULcxaAEWrFqvtur2Qtc4TJlQ-PSJrJ9c2_NeHhXhW-g,3125
dataservice/exceptions.py,sha256=2HqdX1gEM9SbLYaQX_Gyl3HnUkeQqeg0Y9ZNMA3xRUE,634
dataservice/files.py,sha256=ORiRtDtYBLaGnRQoXctOkMDm7q-C1HCnNxksihb6SBk,1999
dataservice/logs.py,sha256=2POOuBvIhHnp6R7e8eXRjt9mHPJthrj41cmRieXMmmU,1311
dataservice/models.py,sha256=b8G4dGSHKxLoGE3_lAmdHOASa_eE_k8--8sacnK2S1Y,4940
dataservice/pw_draft.py,sha256=W8FXov1feGT-_EPHarT_Lja-LK3C-GhWWkVOUmhUFGU,364
dataservice/service.py,sha256=tm8wjv9X4Kl3PNTcFBwLP7Z5xS_lkkuMmjQVFYSoEt0,2832
dataservice/worker.py,sha256=dKyZjqx-g_Y7KQvImxCD4MbfN9bI9VWaKAG1olVpkfE,12499
python_dataservice-0.1.0.dist-info/LICENSE,sha256=YFuh1Fs-GEiQo-dmT_kAULdSQp9PNIda3yXzZIvCs8w,1068
python_dataservice-0.1.0.dist-info/METADATA,sha256=TklXRY8V0Ml9qHmczOeSQkSlyea-ylkUrWPAOTmqsYI,3800
python_dataservice-0.1.0.dist-info/WHEEL,sha256=sP946D7jFCHeNz5Iq4fL4Lu-PrWrFsgfLXbbkciIZwg,88
python_dataservice-0.1.0.dist-info/RECORD,,
